apps/terran/api/example-terran-resource.yaml

```
@@ -0,0 +1,76 @@
+metadata:
+  name: example-app
+  namespace: default
+spec:
+  kcl_config:
+    source:
+      type: inline
+      script: |
+        # KCL script to generate Kubernetes resources
+        import k8s.api.core.v1 as corev1
+        import k8s.api.apps.v1 as appsv1
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)